### PR TITLE
docs: sd-spinner docs [skip chromatic] 

### PR DESCRIPTION
**Pull request by auroraVasconcelos**
## Definition of Reviewable:

- [ ] Documentation is created/updated
- [ ] Migration Guide is created/updated
- [ ] E2E tests (features, a11y, bug fixes) are created/updated
- [ ] Stories (features, a11y) are created/updated
- [ ] relevant tickets are linked

### CODE REVIEW

**Comment by coraliefeil** [comment]
I’ll leave the design approval to you @MartaPintoTeixeira

File: packages/components/src/components/spinner/spinner.stories.ts [hunk 1]
```diff
@@ -1,75 +1,69 @@
...
-  render: (args: any) => {
-    return generateTemplate({ args });
-  }
+  name: 'Default',
+  render: () => html`<sd-spinner></sd-spinner>`
 };
 
 /**
- * Use the `color` attribute to change the color of the spinner and correspond (currentColor) with the parents color.
+ * Offers 4 `color` variants for alternate experiences:
```
Comment (MartaPintoTeixeira) [inline packages/components/src/components/spinner/spinner.stories.ts:28]: Offers “color” variants for alternate experiences:

Comment (auroraVasconcelos) [reply packages/components/src/components/spinner/spinner.stories.ts]: Done

File: packages/components/src/components/spinner/spinner.stories.ts [hunk 2]
```diff
@@ -1,75 +1,69 @@
...
- * Default: This shows sd-spinner in its default state.
- */
-
 export const Default = {
-  render: (args: any) => {
-    return generateTemplate({ args });
-  }
+  name: 'Default',
+  render: () => html`<sd-spinner></sd-spinner>`
 };
```
Comment (mariohamann) [inline packages/components/src/components/spinner/spinner.stories.ts:25]: Be careful: The first story still has to be interactive. Please revert.

File: packages/components/src/components/spinner/spinner.stories.ts [hunk 3]
```diff
@@ -1,75 +1,69 @@
...
+      </div>
+      <div class="bg-primary p-4">
+        <sd-spinner color="white"></sd-spinner>
+      </div>
+    </div>`
 };
 
 /**
- * Use the font-size in css to scale the spinner.
+ * Use the `font-size` in css to scale the spinner proportionally.
```
Comment (mariohamann) [inline packages/components/src/components/spinner/spinner.stories.ts:52]:
```suggestion
 * Use the `font-size` in CSS to scale the spinner proportionally.
```

**Review by mariohamann — CHANGES_REQUESTED** [review]

File: packages/components/src/components/spinner/spinner.stories.ts [hunk 4]
```diff
@@ -1,75 +1,69 @@
...
+    html`<div class="flex flex-row gap-12">
+      <div style="font-size: inherit">
+        <sd-spinner color="currentColor"></sd-spinner>
+      </div>
+      <div style="font-size: 2rem">
+        <sd-spinner color="currentColor"></sd-spinner>
+      </div>
+      <div style="font-size: 4rem">
+        <sd-spinner color="currentColor"></sd-spinner>
+      </div>
```
Comment (yoezlem) [inline packages/components/src/components/spinner/spinner.stories.ts:67]: In Figma the size story has color="primary"
image

File: packages/components/src/components/spinner/spinner.stories.ts [hunk 5]
```diff
@@ -1,75 +1,69 @@
...
-          alternate: 'x',
-          colors: ['', 'rgb(var(--sd-color-primary, 0 53 142))', '#F6F6F6']
-        }
-      },
-      args
-    });
-  }
+  render: () =>
+    html` <div class="flex flex-row items-center gap-12">
+      <div class="bg-neutral-100 p-4">
```
Comment (yoezlem) [inline packages/components/src/components/spinner/spinner.stories.ts:39]: Has it been agreed with design that `bg-neutral` should be added to `currentColor`?

File: packages/components/src/components/spinner/spinner.stories.ts (hunk 4 above)
Comment (auroraVasconcelos) [reply packages/components/src/components/spinner/spinner.stories.ts:67]: The primary is the default color in figma, but not the default color in code. Currently, I used the default color but I can make it primary if it makes more sense

**Review by MartaPintoTeixeira — APPROVED** [review]
looking good

File: packages/components/src/components/spinner/spinner.stories.ts (hunk 4 above)
Comment (auroraVasconcelos) [reply packages/components/src/components/spinner/spinner.stories.ts]: Updated

File: packages/components/src/components/spinner/spinner.stories.ts (hunk 5 above)
Comment (auroraVasconcelos) [reply packages/components/src/components/spinner/spinner.stories.ts]: Discussed with Design and agreed to remove the background

File: packages/components/src/components/spinner/spinner.stories.ts [hunk 6]
```diff
@@ -1,75 +1,66 @@
...
 export const Default = {
   render: (args: any) => {
     return generateTemplate({ args });
   }
 };
 
 /**
- * Use the `color` attribute to change the color of the spinner and correspond (currentColor) with the parents color.
+ * Offers `color` variants for alternate experiences:
+ * - `currentColor` (default): used to inherit the color from the parent element
```
Comment (mariohamann) [inline packages/components/src/components/spinner/spinner.stories.ts:29]:
```suggestion
 * - `currentColor` (default): used to inherit the `text-color` from the parent element
```

**Pull request merged**